templates/solutions/ubuntu-os/index.html

```diff
@@ -0,0 +1,653 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1X3osSqcm3rWLqYUJr5G4fXEEHCby-j5FEJwClLNx5S4/edit
+{% endblock meta_copydoc %}
+
+{% block title %}The world's favourite enterprise Linux platform{% endblock %}
+
+{% block meta_description %}
+  Ubuntu is the world's most popular enterprise Linux distribution. Unify your tech stack with the enterprise OS that is favoured for servers, IoT and applications.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Innovate with confidence.</h1>
+        <h2>Choose the world's favourite enterprise Linux.</h2>
+        <p class="p-section--shallow">
+          Bring harmony to your stack from the desktop to the edge with Ubuntu, the OS that successful enterprises have in common. Ubuntu brings standardisation and stability  together, offering you a platform for innovation, combined with the freedom that transparent, open source code offers.
+        </p>
+        <hr class="is-muted" />
```

Comment:
I've been involved in [another conversation](https://chat.canonical.com/canonical/pl/dy1mx1hizbnjxkh44ac3nr8dxo) around the use of 'p-rule'. As far as I understand we should move towards using 'p-rule--muted', as this follows BEM convention and allows easier changes in the future.

Reply:
I updated all hrs that were using `is-muted`, but it looks like in this and similar instances `is-muted` should be used instead as we need a bit of margin to keep the separator and button from touching:
![image](https://github.com/user-attachments/assets/7b39b497-88e0-4cc4-a915-0b0faf06e8a2)